packages/babel-preset-flow/package.json

```diff
@@ -13,6 +13,7 @@
     "types"
   ],
   "dependencies": {
-    "babel-plugin-transform-flow-strip-types": "^6.22.0"
+    "babel-plugin-transform-flow-strip-types": "^6.22.0",
+    "babel-plugin-transform-class-properties": ">6.24.1"
```

Comment:
Using `>` is a no-go because that would also match `7.x` versions in the future.